xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/less/edit.less

```diff
@@ -7,6 +7,9 @@
   .buttons {
     margin: 0;
     padding: 1em;
+    #autosaveControl.disabled{
```

Comment:
Missing a space before { (I don't know if we have a checkstyle for those, but doesn't look neat :)

Reply:
:+1: Addressed in [a7ba0e4](https://github.com/xwiki/xwiki-platform/pull/2170/commits/a7ba0e4f83468e4216e074f6611cf3029cc4fefb)